static/js/containers/VideoDetailPage.js

```diff
@@ -55,6 +60,41 @@ class VideoDetailPage extends React.Component {
     dispatch(setDrawerOpen(open));
   };
 
+  deleteSubtitle = async (videoSubtitleId) => {
+    const {
+      dispatch,
+      videoKey
+    } = this.props;
+    await dispatch(actions.videoSubtitles.delete(videoSubtitleId));
+    dispatch(actions.videos.get(videoKey));
+  };
+
+  uploadVideoSubtitle = async () => {
+    const {
+      dispatch,
+      video,
+      videoKey,
+      videoSubtitleUi: {videoSubtitleForm}
+    } = this.props;
+    const formData = new FormData();
```

Comment:
This variable only needs to be instantiated within the `if` statement

Reply:
OK